README.md

```diff
@@ -89,9 +98,84 @@ Below are instructions how to get the different tokens and keys.
 **Try it out, step 3/3: Start the complete stack using [docker-compose](https://docs.docker.com/compose/)**
 
 ```bash
-docker-compose -f docker-compose.yml -f docker-compose.prod.yml up --build
+docker-compose --file docker-compose.yml --file docker-compose.prod.yml up --build
 ```
 
+# For developers
+
+## General workflow when making changes
+
+Let's say you followed the steps above, and have a running instance of the
+Research Software Directory. Now you may want to make some changes to bring the
+frontend in line with your institute's branding. For example, you could follow
+the steps outlined [here](faq/how-do-i-change-the-font.md) to change the fonts.
+
+Now the question is, after making your changes, how do you get to see them?
+Here's how:
+
+1. Go to the terminal where you started ``docker-compose``
+1. Use Ctrl+C to stop the running instance of Research Software Directory
+1. Check which docker containers you have with:
+
+    ```
+    docker ps -a
+    ```
+
+    for example, mine says:
+
+    ```
+    CONTAINER ID        IMAGE                  COMMAND                   CREATED             STATUS                       PORTS               NAMES
+    a570e0cc8ea8        certbot/certbot        "sh /certbot.sh"          3 minutes ago       Exited (1) 3 minutes ago                         tmpufoyhj4z6k_certbot_1
+    b15cabf82f23        rsdnlesc/nginx_proxy   "/bin/sh -c 'sh /sta…"    3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_nginx_1
+    5e7ab363a5ba        rsdnlesc/frontend      "/bin/sh -c 'sh -c \"…"   3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_frontend_1
+    b7b24d3ce94b        rsdnlesc/tasks         "/bin/sh -c 'crond -…"    3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_tasks_1
+    8703e87e709f        rsdnlesc/backend       "/bin/sh -c 'gunicor…"    3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_backend_1
+    531a6d53430b        rsdnlesc/admin         "sh -c 'rm -rf /buil…"    3 minutes ago       Exited (0) 3 minutes ago                         tmpufoyhj4z6k_admin_1
+    d90006bc24ff        mongo:3.6              "/mongo.sh --bind_ip…"    3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_mongo_1
+    34739f47c066        rsdnlesc/auth-github   "/bin/sh -c 'gunicor…"    3 minutes ago       Exited (137) 7 seconds ago                       tmpufoyhj4z6k_auth_1
+    ```
+
+    Use ``docker rm`` to delete the container you want to update, e.g. the frontend container:
+
+    ```
+    docker rm tmpufoyhj4z6k_frontend_1
+    ```
+
+    Check which images you have with ``docker images``, remove by ID as appropriate with ``docker rmi <IMAGE ID>``:
```

Comment:
Instead of using the <IMAGE ID> use the repository name.
So for frontend:
```
docker-compose rm --force frontend
docker rmi --force rsdnlesc/frontend
```